examples/first/cli.js

```diff
@@ -0,0 +1,3 @@
+export function first(args) {
+  console.log('I DO STUFF YEAH>', args);
+}
```

Comment:
I totally get how this is tied together with the example but there are some things that are mystery that could probably be shown here for people diving in. Mainly, how do I get those args and how do I do something with them. That would cover the boilerplate most people need to get something working.